tests/conftest.py

```diff
@@ -0,0 +1,238 @@
+"""Module containing helper functions for testing PyTeal Utils."""
```

Comment:
Since this is library code, I recommend not using `test` in the filename. As much as I dread the overused `helpers` or `utils` those are better than `*test` (and maybe you have an even better name).

Reply:
Something about this filename is special, I was only able to import from it if it was named `conftest.py`

Reply:
Maybe adding an `__init__.py` at the highest level will solve it? I'm inspired by this [S.O. answer](https://stackoverflow.com/questions/54895002/modulenotfounderror-with-pytest). If not, it's not worth further hassle so let's stick to that name.